### PR TITLE
Fix null pointer when showing locale claim with invalid locale format

### PR DESCRIPTION
**Pull request by bhagyasakalanka**
### Purpose
Fix null pointer when showing locale claim with invalid locale format

### Related PRs
- https://github.com/wso2/identity-apps/pull/5980

### CODE REVIEW

**Review by pavinduLakshan — APPROVED** [review]

File: features/admin.users.v1/components/user-profile.tsx [hunk 1]
```diff
@@ -525,18 +525,42 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
...
         }
 
-        let [ language, region ] = locale.split(/[-_]/);
+        const separatorIndex: number = localeString.search(/[-_]/);
 
-        language = language.toLowerCase();
-        region = region.toUpperCase();
+        let language: string;
+        let region: string;
+        let normalizedLocale: string;
```
Comment (Achintha444) [inline features/admin.users.v1/components/user-profile.tsx:543]:
```suggestion
        let normalizedLocale: string = localString
```

and remove the else block

File: features/admin.users.v1/components/user-profile.tsx [hunk 2]
```diff
@@ -525,18 +525,42 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
...
+        if (separatorIndex !== -1) {
+            language = localeString.substring(0, separatorIndex).toLowerCase();
+            region = localeString.substring(separatorIndex + 1).toUpperCase();
+            normalizedLocale = `${language}${localeJoiningSymbol}${region}`;
+        } else {
+            normalizedLocale = localeString;
+        }
+
+        if (updateSupportedLanguage) {
+            if (!supportedI18nLanguages[normalizedLocale]) {
```
Comment (Achintha444) [inline features/admin.users.v1/components/user-profile.tsx:554]:
```suggestion
        if (updateSupportedLanguage && !supportedI18nLanguages[normalizedLocale]) {
```

File: features/admin.users.v1/components/user-profile.tsx (hunk 1 above)
Comment (bhagyasakalanka) [reply features/admin.users.v1/components/user-profile.tsx]: Fixed

File: features/admin.users.v1/components/user-profile.tsx (hunk 2 above)
Comment (bhagyasakalanka) [reply features/admin.users.v1/components/user-profile.tsx]: Fixed: 503184c093030f2b3969b8b6d0f116d490340907

File: features/admin.users.v1/components/user-profile.tsx [hunk 3]
```diff
@@ -525,18 +525,42 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
...
+            return localeString;
         }
 
-        let [ language, region ] = locale.split(/[-_]/);
+        const separatorIndex: number = localeString.search(/[-_]/);
 
-        language = language.toLowerCase();
-        region = region.toUpperCase();
+        let language: string;
+        let region: string;
```
[the remark this answers is not in the document]
Comment (bhagyasakalanka) [reply features/admin.users.v1/components/user-profile.tsx]: Fixed: 503184c093030f2b3969b8b6d0f116d490340907

**Review by Achintha444 — APPROVED** [review]

**Pull request merged**